qubesdbus/domain_manager.py

```diff
@@ -59,15 +57,17 @@ class DomainManager(PropertiesObject):
                properties
     '''
 
-    def __init__(self, qubes_data: DBusProperties, domains: List[DBusProperties]) -> None:
+    def __init__(self) -> None:
+        self.app = qubesadmin.Qubes()
```

Comment:
DBusServiceObject already have self.app set. Since you need it here before `super()` call, maybe make it conditional in DBusServiceObject (`if not hasattr(self, 'app')`)?